pages/about.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+
+const AboutPage = () => {
+  return (
+    <div className="bg-black z-20 text-white max-w-full fixed inset-0 overflow-y-auto overflow-x-auto">
+      <nav className="flex justify-between ml-8 mr-8 mt-3 h-12">
+        <Link href="/">Atlas of Ownership</Link>
+        <Link href="/" className="text-lg">X</Link>
```

Comment:
nit: We use the `<Close size={32} className="cursor-pointer" />` Carbon Icon for this on the entry page